eth/v1alpha1/beacon_chain.proto

```diff
@@ -405,6 +397,10 @@ message ListBlocksResponse {
     int32 total_size = 3;
 }
 
+message StreamBlocksRequest {
+    bool verified_only = 1;
```

Comment:
Let's add some commentary here. i.e. this is a request to only return blocks with valid signatures. 
This proto file is a self-documenting file so we should try to add comments when we can.